### ISSUE TITLE
Notification settings link opens up a blank screen in a new window on 0.23.0

### ISSUE THREAD

**Issue by arudnev** [nwjs/nw.js#5948]
Similar to #4582, but now with new macOS notification center integrated implementation the notifications show the settings button that leads nowhere.

To reproduce, execute the following in dev console.

```javascript
new Notification('title', {body: 'body'})
```

and then hover over native notifications, click on Settings button there.

**Comment by Christywl** [comment]
I can reproduce this issue on Mac with nwjs-sdk-v0.23.0. Click setting button and open a blank window.
(There is no Settings button on Windows and Linux)

**Comment by arudnev** [comment]
Guys, is there a way to disable native notifications until this issue is fixed? I can get to the old behavior by executing the following and then manually disabling native notifications, but can not figure out a way to do it programmatically / via configuration settings.

```javascript
require('nw.gui').Window.open('chrome://flags/#enable-native-notifications')
```

**Comment by rogerwang** [comment]
Will fix this soon with upcoming 0.23.4

**Comment by rogerwang** [comment]
This is fixed in git and will be available in the next nightly build.
